_posts/2020-10-06-bazel-3-6.md

```diff
@@ -0,0 +1,95 @@
+    This may cause slight breakages depending on the content of the prelude
+    file. For more details, see the discussion in [issue
+    #11940](https://github.com/bazelbuild/bazel/issues/11940).
+*   The flag `--experimental_ui_limit_console_output` is removed. If you
+    use `--experimental_ui_limit_console_output=1` to silence terminal
+    output, you should use `--ui_event_filters=` instead.
+*   The flag `--proto:instantiation_stack` must be enabled with
+    `--record_rule_instantiation_callstack` to see call stack in proto output
+    from Bazel query.
+*   A few other flags are removed:
+    `--experimental_ignore_deprecated_instrumentation_spec`,
+    `--experimental_transparent_compression`, `--experimental_action_args`,
+    `--incompatible_no_support_tools_in_action_inputs`.
+
+## Android
+
+*   NDK 21 is now supported.
+*   Building Android apps for legacy multidex (pre-L) now requires a main-dex
+    list if the application does not fit into a single DEX file.
+
+## C++
+
+*   [`cc_common.compile`](https://docs.bazel.build/versions/master/skylark/lib/cc_common.html#compile)
+    now supports `include_prefix` and `strip_include_prefix`.
+
+
+## External repositories
+
+*   Stop needlessly parsing WORKSPACE files from external repositories.
+*   Dot (`.`) is now allowed in workspace names. See [issue #11837](https://github.com/bazelbuild/bazel/issues/11837).
+
+
+## Starlark
+
+*   In Starlark, the Args object supports a new parameter file format
+    `flag_per_line`, compatible with the Abseil flags library.
+*   The Starlark interpreter now correctly emits an error if the operand of the
+	first loop in a list comprehension refers to a variable bound by a later
+	loop, such as y in this example:
+	```python
+    [e1 for x in f(**y**) in e2 for y in e3] # error: undefined y`
+    ```
+    This may cause latent dynamic errors to become static errors.
+
+
+## Execution
+
+*   Added the `--experimental_worker_max_multiplex_instances` flag to configure
+    the number of WorkRequests that are sent concurrently to one multiplex
+    worker process. This flag has the same syntax as the
+    `--worker-max-instances` flag. The `--worker_max_instances` flag will no
+    longer be used to determine max instances for multiplex workers, because the
+    two have different resource requirements. Multiplex workers have an
+    8-instance maximum, by default.
+*   Added the `--experimental_worker_multiplex` flag, which is enabled by
+    default. When enabled, multiplex workers are used for all mnemonics that
+    support them.
+*   Added support for a `supports-graceful-termination` execution requirement
+    and tag, which causes Bazel to send a SIGTERM to any tagged actions before
+    sending a delayed SIGKILL. This is to give actions, and more specifically
+    tests, a chance to clean up after themselves.
+
+
+## Other changes
+
+*   Bazel will now skip printing action stdout/stderr contents if they exceed
```

Comment:
now skips

Reply:
Done, thanks.
For consistency, I've also removed a future above ("will no longer be used").